<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadGraph</class>
 <widget class="QMainWindow" name="LoadGraph">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load a Graph</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="testBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Lancer</string>
    </property>
   </widget>
   <widget class="QLabel" name="filenameLbl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>Entrer le nom du fichier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filenameLineEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>151</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="cursorPosition">
     <number>0</number>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>70</y>
      <width>81</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="grapheArea">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>2</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>CLIQUE</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicView_graphe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>411</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>409</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_newClick"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_newClick">
   <property name="text">
    <string>New Clique</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>testBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>testBtn</sender>
   <signal>clicked()</signal>
   <receiver>LoadGraph</receiver>
   <slot>launchProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>23</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>LoadGraph</receiver>
   <slot>loadFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>launchProcess()</slot>
  <slot>loadFile()</slot>
 </slots>
</ui>
